app/definitions/rest/v1/teams.ts

```diff
@@ -4,4 +4,12 @@ export type TeamsEndpoints = {
 	'teams.removeRoom': {
 		POST: (params: { roomId: string; teamId: string }) => { room: IRoom };
 	};
+	'teams.listRoomsOfUser': {
+		GET: (params: { teamId: string; userId: string }) => {
+			rooms: IRoom[];
```

Comment:
Are you sure the return of this api is `IRoom`?
I think it's something like `IServerRoomItem`, given that `IRoom` is our implementation that merges the type to look like the room after the merge.